website/docs/d/kms_secrets.html.markdown

```diff
@@ -22,6 +22,8 @@ If you do not already have a `CiphertextBlob` from encrypting a KMS secret, you
 $ echo -n 'master-password' > plaintext-password
 $ aws kms encrypt --key-id ab123456-c012-4567-890a-deadbeef123 --plaintext fileb://plaintext-password --encryption-context foo=bar --output text --query CiphertextBlob
 AQECAHgaPa0J8WadplGCqqVAr4HNvDaFSQ+NaiwIBhmm6qDSFwAAAGIwYAYJKoZIhvcNAQcGoFMwUQIBADBMBgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDI+LoLdvYv8l41OhAAIBEIAfx49FFJCLeYrkfMfAw6XlnxP23MmDBdqP8dPp28OoAQ==
+aws kms encrypt --key-id ab123456-c012-4567-890a-deadbeef123 --plaintext fileb://plaintext-password --encryption-algorithm RSAES_OAEP_SHA_256 --output text --query CiphertextBlob
```

Comment:
Perhaps we can add a helpful comment above this command describing what it does and why it is different from the one above. For example, here I see that it uses `RSAES_OAEP_SHA_256`. An example of this formatting would be the following below. Also, we add the `$` sign at the start of the command for consistency with the ones above

```suggestion
# Helpful comment
$ aws kms encrypt --key-id ab123456-c012-4567-890a-deadbeef123 --plaintext fileb://plaintext-password --encryption-algorithm RSAES_OAEP_SHA_256 --output text --query CiphertextBlob

Reply:
Done as requested.